stats/src/main/java/io/airlift/stats/QuantileDigest.java

```diff
@@ -257,12 +257,11 @@ public void add(long value)
     /**
      * Adds a value to this digest. The value must be {@code >= 0}
      */
-    public void add(long value, long count)
+    public void add(long value, double weight)
     {
         checkArgument(count > 0, "count must be > 0");
```

Comment:
There's a compilation failure. You'll need to rename this to `weight`, also.

Line 272 also needs to change from:

```java
weight = weight(nowInSeconds) * count;
```

to 

```java
weight *= weight(nowInSeconds);
```

[ERROR] /home/travis/build/airlift/airlift/stats/src/main/java/io/airlift/stats/QuantileDigest.java:[262,22] error: cannot find symbol
  symbol:   variable count
  location: class QuantileDigest
[ERROR] /home/travis/build/airlift/airlift/stats/src/main/java/io/airlift/stats/QuantileDigest.java:[272,44] error: cannot find symbol
  symbol:   variable count
  location: class QuantileDigest
```